include/common/dm.h

```diff
@@ -82,6 +83,14 @@ struct device *dm_get_by_class(uint32_t class);
  */
 struct device *dm_get_by_name(const char *name);
 
+/**
+ * Sets the mode of the GPIO pins specified for a device.
```

Comment:
Please be affirmative, not descriptive: "Sets" → "Set".

Also, please put the prototype in the header in alphabetical order

Reply:
Done.